doc/connectivity/bluetooth/api/audio/shell/tbs.rst

```diff
@@ -0,0 +1,238 @@
+the telephone bearers on the server. If the server has both GTBS and TBS,
+the client may subscribe and use either when sending requests if
+:code:`BT_TBS_CLIENT_GTBS` is enabled.
+
+.. code-block:: console
+
+   tbs_client --help
+   tbs_client - Bluetooth TBS_CLIENT shell commands
+   Subcommands:
+      discover                       :Discover TBS [subscribe]
+      set_signal_reporting_interval  :Set the signal reporting interval
+                                       [<{instance_index, gtbs}>] <interval>
+      originate                      :Originate a call [<{instance_index, gtbs}>]
+                                       <uri>
+      terminate                      :terminate a call [<{instance_index, gtbs}>]
+                                       <id>
+      accept                         :Accept a call [<{instance_index, gtbs}>] <id>
+      hold                           :Place a call on hold [<{instance_index,
+                                       gtbs}>] <id>
+      retrieve                       :Retrieve a held call [<{instance_index,
+                                       gtbs}>] <id>
+      read_provider_name             :Read the bearer name [<{instance_index,
+                                       gtbs}>]
+      read_bearer_uci                :Read the bearer UCI [<{instance_index, gtbs}>]
+      read_technology                :Read the bearer technology [<{instance_index,
+                                       gtbs}>]
+      read_uri_list                  :Read the bearer's supported URI list
+                                       [<{instance_index, gtbs}>]
+      read_signal_strength           :Read the bearer signal strength
+                                       [<{instance_index, gtbs}>]
+      read_signal_interval           :Read the bearer signal strength reporting
+                                       interval [<{instance_index, gtbs}>]
+      read_current_calls             :Read the current calls [<{instance_index,
+                                       gtbs}>]
+      read_ccid                      :Read the CCID [<{instance_index, gtbs}>]
+      read_status_flags              :Read the in feature and status value
+                                       [<{instance_index, gtbs}>]
+      read_uri                       :Read the incoming call target URI
+                                       [<{instance_index, gtbs}>]
+      read_call_state                :Read the call state [<{instance_index, gtbs}>]
+      read_remote_uri                :Read the incoming remote URI
+                                       [<{instance_index, gtbs}>]
+      read_friendly_name             :Read the friendly name of an incoming call
+                                       [<{instance_index, gtbs}>]
+      read_optional_opcodes          :Read the optional opcodes [<{instance_index,
+                                       gtbs}>]
+
+
+In the following examples, notifications from GTBS is ignored, unless otherwise
```

Comment:
This was also wrong in the original document, but suggest to fix it here
 ```suggestion
In the following examples, notifications from GTBS are ignored, unless otherwise
```